tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/TsFileMetaData.java

```diff
@@ -1,306 +1,388 @@
-      return null;
-    }
-  }
-
-  public Map<String, MeasurementSchema> getMeasurementSchema() {
-    return measurementSchema;
-  }
-
-  /**
-   * use the given outputStream to serialize.
-   *
-   * @param outputStream -output stream to determine byte length
-   * @return -byte length
-   */
-  public int serializeTo(OutputStream outputStream) throws IOException {
-    int byteLen = 0;
-
-    byteLen += ReadWriteIOUtils.write(deviceIndexMap.size(), outputStream);
-    for (Map.Entry<String, TsDeviceMetadataIndex> entry : deviceIndexMap.entrySet()) {
-      byteLen += ReadWriteIOUtils.write(entry.getKey(), outputStream);
-      byteLen += entry.getValue().serializeTo(outputStream);
-    }
-
-    byteLen += ReadWriteIOUtils.write(measurementSchema.size(), outputStream);
-    for (Map.Entry<String, MeasurementSchema> entry : measurementSchema.entrySet()) {
-      byteLen += ReadWriteIOUtils.write(entry.getKey(), outputStream);
-      byteLen += entry.getValue().serializeTo(outputStream);
-    }
-
-    byteLen += ReadWriteIOUtils.writeIsNull(createdBy, outputStream);
-    if (createdBy != null) {
-      byteLen += ReadWriteIOUtils.write(createdBy, outputStream);
-    }
-
-    byteLen += ReadWriteIOUtils.write(totalChunkNum, outputStream);
-    byteLen += ReadWriteIOUtils.write(invalidChunkNum, outputStream);
-
-    return byteLen;
-  }
-
-  /**
-   * use the given buffer to serialize.
-   *
-   * @param buffer -buffer to determine byte length
-   * @return -byte length
-   */
-  public int serializeTo(ByteBuffer buffer) throws IOException {
-    int byteLen = 0;
-
-    byteLen += ReadWriteIOUtils.write(deviceIndexMap.size(), buffer);
-    for (Map.Entry<String, TsDeviceMetadataIndex> entry : deviceIndexMap.entrySet()) {
-      byteLen += ReadWriteIOUtils.write(entry.getKey(), buffer);
-      byteLen += entry.getValue().serializeTo(buffer);
-    }
-
-    byteLen += ReadWriteIOUtils.write(measurementSchema.size(), buffer);
-    for (Map.Entry<String, MeasurementSchema> entry : measurementSchema.entrySet()) {
-      byteLen += ReadWriteIOUtils.write(entry.getKey(), buffer);
-      byteLen += entry.getValue().serializeTo(buffer);
-    }
-
-    byteLen += ReadWriteIOUtils.writeIsNull(createdBy, buffer);
-    if (createdBy != null) {
-      byteLen += ReadWriteIOUtils.write(createdBy, buffer);
-    }
-
-    byteLen += ReadWriteIOUtils.write(totalChunkNum, buffer);
-    byteLen += ReadWriteIOUtils.write(invalidChunkNum, buffer);
-
-    return byteLen;
-  }
-
-  public int getTotalChunkNum() {
-    return totalChunkNum;
-  }
-
-  public void setTotalChunkNum(int totalChunkNum) {
-    this.totalChunkNum = totalChunkNum;
-  }
-
-  public int getInvalidChunkNum() {
-    return invalidChunkNum;
-  }
-
-  public void setInvalidChunkNum(int invalidChunkNum) {
-    this.invalidChunkNum = invalidChunkNum;
-  }
-
-  public List<MeasurementSchema> getMeasurementSchemaList() {
-    return new ArrayList<MeasurementSchema>(measurementSchema.values());
-  }
-}
+/**

```

Comment:
I remember that we have changed `/**` to `/*`

Reply:
I will fix it